src/pods/edit-table/components/commands/commands.component.tsx

```diff
@@ -12,6 +12,7 @@ interface Props {
   fields: FieldVm[];
   onMoveDownField: (fieldId: GUID) => void;
   onMoveUpField: (fieldId: GUID) => void;
+  DeleteIsVisible: boolean;
```

Comment:
I would rather name it:

`IsDeleteVisible` (verb first) or `showDelete`

Which one do you preffer?

Reply:
I would rather name it: isDeleteVisible